docs/learn-about-neptune/team-management.rst

```diff
@@ -105,6 +106,5 @@ Add users to a project
 
 .. note::
 
-    - If the project is created in a team organization, the user is assigned to the project immediately and notified by email.
-    - If the project is created in an individual account, you can only add users to a public project. The user is notified by email and must accept the invitation to be added.
-    You cannot add users to a private project.
\ No newline at end of file
+    - The user is invited to the project by email. The user needs to accept the invitation, before that user appears in the list.
+    - If the project is created in an individual account, you can only add users to a public project. The user is notified by email and must accept the invitation to be added. You cannot add users to a private project.
```

Comment:
you can only add users -> you can only invite users
You cannot add users -> You cannot invite users